spec/factories/games.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :game do
+    title { Faker::Game.title }
+    game_progress { 10.00 }
```

Comment:
same as with the user - maybe there should be a trait `:with_progress` or maybe even `:with_progress_10`, `:with_progress_50` etc?